### PR TITLE
Enable coverage reports from codecov

### PR DESCRIPTION
**Pull request by NickeZ**

### CODE REVIEW

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ignore:
+  - 'test'
+  - 'external'
+  - 'src/drivers/**/*'
```
Comment (benma) [inline .codecov.yml]: also `tools`?

File: .ci/travis-ci [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
 git fetch
 
-docker run -v ${TRAVIS_BUILD_DIR}:/bitbox02-firmware/ \
-       -i shiftcrypto/firmware_v2:2 \
-       bash -c "make -C /bitbox02-firmware ci"
+docker run \
+	--cap-add SYS_PTRACE \
+	--volume ${TRAVIS_BUILD_DIR}:/bitbox02-firmware/ \
+	--workdir /bitbox02-firmware \
+	shiftcrypto/firmware_v2:2 \
```
Comment (benma) [inline .ci/travis-ci]: can this be in a variable so you can't forget to update the tag in the 2nd place?

File: .codecov.yml (hunk 1 above)
Comment (NickeZ) [reply .codecov.yml]: AFAIK we don't compile any code in tools, so not needed for now.

File: .ci/travis-ci (hunk 2 above)
Comment (NickeZ) [reply .ci/travis-ci]: ack

File: .codecov.yml (hunk 1 above)
Comment (benma) [reply .codecov.yml]: turns out we do, nanopb encoding/decoding ;)
fix here: https://github.com/digitalbitbox/bitbox02-firmware/pull/47

**Pull request merged**